### PR TITLE
[script.module.pyxbmct] 1.3.1+matrix

### PR DESCRIPTION
**Pull request by romanvm**
### Add-on details:

- **General**
  - Add-on name: PyXBMCt
  - Add-on ID: script.module.pyxbmct
  - Version number: 1.3.1+matrix
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/romanvm/script.module.pyxbmct
  
PyXBMCt is a mini-framework for simple XBMC addon UI buliding. It is similar to PyQt and provides parent windows, a number of UI controls (widgets) and a grid layout manager to place controls.

### Description of changes:

1.3.1:
- Fix incompatibility with newer Python 3 Kodi builds
- Replace future library with six for Python 3 compatibility

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: script.module.pyxbmct/changelog.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[B]Version 1.3.0[/B]
```
Comment (basrieter) [inline script.module.pyxbmct/changelog.txt]: Please exclude the changelog.txt from the submission and use the `<news>` tag in addon.xml for the changes in the latest version.

File: script.module.pyxbmct/addon.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.six" />
+    <import addon="script.module.kodi-six" />
+  </requires>
+  <extension point="xbmc.python.module" library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en_GB">PyXBMCt UI framework</summary>
+    <description lang="en_GB">PyXBMCt is a mini-framework for simple XBMC addon UI buliding. It is similar to PyQt and provides parent windows, a number of UI controls (widgets) and a grid layout manager to place controls.</description>
+    <license>GNU GPL v.3</license>
```
Comment (basrieter) [inline script.module.pyxbmct/addon.xml]: Perhaps use the SPDX identifier: GPL-3.0-only

**Comment by enen92** [comment]
It's fine too keep it to ease the migration. That way only addon.xml is needed

**Comment by basrieter** [comment]
> It's fine too keep it to ease the migration. That way only addon.xml is needed

Ok, so then it's just the changelog.txt and perhaps the SPDX in the addon.xml

File: script.module.pyxbmct/addon.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyxbmct"
+   name="PyXBMCt"
+   version="1.3.1+matrix"
```
Comment (enen92) [inline script.module.pyxbmct/addon.xml]: `1.3.1+matrix.1` if possible, just for consistency with other script modules already on matrix

File: script.module.pyxbmct/addon.xml (hunk 2 above)
Comment (enen92) [inline script.module.pyxbmct/addon.xml]: spdx: https://spdx.org/licenses/

**Comment by romanvm** [comment]
Updated

**Pull request merged**